infra/k8s/greptime/eks-deployment-runbook.md

```
@@ -41,50 +41,66 @@ helm upgrade --install greptimedb-operator greptime/greptimedb-operator -n grept
 
 ## 4) Deploy Greptime cluster (Helm)
 
-Replace the `<PLACEHOLDER>` values in `greptime-values.yaml` with your Aurora host, database, S3 bucket, region, and prefix before running.
+Replace the `<PLACEHOLDER>` values in `greptime-values.yaml` with your Aurora host, database, S3 bucket, and region before running.
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)